frontend/src/types/vendor.ts

```diff
@@ -36,6 +44,7 @@ export type Vendor = Pick<BackendVendor, 'id'
   'state': string | null,
   'instagram': string | null,
   'google_maps_place': string | null,
+  'testimonials': VendorTestimonial[]
```

Comment:
if we're using the vendor field without transformation, you can use the BackendVendor field in Pick<>, instead of a new field

Reply:
Resolving bc testimonials needs to be joined with Vendor!

Reply:
Resolving bc testimonials needs to be joined with Vendor!